editor/src/components/canvas/canvas-strategies/strategies/grid-helpers.ts

```diff
@@ -155,6 +130,7 @@ export function runGridRearrangeMove(
     gridPath,
     targetCellData?.gridCellCoordinates ?? null,
     gridCellCoordinates(row, column),
+    'valid',
   )
 
   switch (moveType) {
```

Comment:
it feels wrong that `runGridMoveRearrange` has to deal with absolute move – in a follow-up PR, we should run the real absolute strategy and remove absolute-related code from here.

Reply:
absolute has been taken out into its own strategy